sos/report/__init__.py

```diff
@@ -922,6 +922,7 @@ def _set_tunables(self):
                         try:
                             val = int(val)
                         except ValueError:
+                            # not a number to convert back to int from argparse
```

Comment:
Maybe we can print here some log as well, for potential diagnosis "why my argument has been ignored"?

Reply:
I don't _think_ that's something to be concerned about here, but would already be logged later on by existing checks. This is the attempt to see "did the user pass something that got auto converted to a string, but we should cast to an int? If not, no actual concern because it should really be a string".